### PR TITLE
COMMERCE-9651 Make Order fields available for use with fragments

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9651

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

**Comment by baharturk** [comment]
Hi @dsitu, I analysed the test that fails with relevant, the locator is changed with this PR and we need to fix a locator. Thnx!

modules > apps > commerce > commerce-product-test > testFunctional > paths > CommerceOrders.path > PENDING_ORDERS_ORDER_ID --> change to be made 'Order ID' instead of 'Oder Id'

**Review by alessiorendina — CHANGES_REQUESTED** [review]
Hey @dsitu ,
everything looks good to me in the code, but @baharturk showed me the result and there are 3 tabs in the order item selector instead of one. Can you take a look at it? Also the PR needs to be rebased cause there are some conflicts. Thank you and well done!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
> Hey @dsitu , everything looks good to me in the code, but @baharturk showed me the result and there are 3 tabs in the order item selector instead of one. Can you take a look at it? Also the PR needs to be rebased cause there are some conflicts. Thank you and well done!

@alessiorendina @baharturk I looked into the 3 tabs issue. Looks like there was a conflict with naming. I had to change the name of the tab to "Orders" as the Object version of Commerce Orders is creating its own tab named "Commerce Orders".

**Comment by dsitu** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/instance/lifecycle/SystemObjectDefinitionMetadataPortalInstanceLifecycleListener.java [hunk 1]
```diff
@@ -236,6 +239,28 @@ private void _apply(
 		}
 	}
 
+	private void _removeItemSelectorViewService() {
```
Comment (andrea-ale-sbarra) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/instance/lifecycle/SystemObjectDefinitionMetadataPortalInstanceLifecycleListener.java:242]: I don't think this is the right approach

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
@dsitu please have a look at this class example BlogsEntryInfoItemSelector it may be that the lack of this is causing the multi tab issue. Please give it a try.

Thanks!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**